.gitignore

```diff
@@ -1,5 +1,34 @@
-node_modules
 
+## MAC files ##
 .DS_Store
 
+## NPM ##
+npm
+npm-debug.log
+node_modules/
+package-lock.json
+bundle.js
 client/dist/bundle.js
+
+pids
+*.pid
+logs
+*.log
+
+# Directory for instrumented libs generated by jscoverage/JSCover
```

Comment:
There's a lot of stuff here: JSCover, Tools like Istanbul, Grunt - are you using all of them?